src/effects/Tracker.h

```diff
@@ -41,50 +41,31 @@
 #include "../Json.h"
 #include "../KeyFrame.h"
 #include "protobuf_messages/trackerdata.pb.h"
+#include "../TrackedObjectBBox.h"
+#include "../Clip.h"
 
-// Tracking info struct
-struct EffectFrameData{
-  size_t frame_id = -1;
-  float rotation = 0;
-  float x1 = -1;
-  float y1 = -1;
-  float x2 = -1;
-  float y2 = -1;
-
-  // Constructors
-  EffectFrameData()
-  {}
-
-  EffectFrameData( int _frame_id)
-  {frame_id = _frame_id;}
-
-  EffectFrameData( int _frame_id , float _rotation, float _x1, float _y1, float _x2, float _y2)
-  {
-      frame_id = _frame_id;
-      rotation = _rotation;
-      x1 = _x1;
-      y1 = _y1;
-      x2 = _x2;
-      y2 = _y2;
-  }
-};
-
-
+using namespace std;
```

Comment:
:no_entry_sign: Never, **ever** in a header file. See #296, which has links to additional references on the issue.
```suggestion
```